pages/contact.html

```diff
@@ -15,16 +15,34 @@
             <img src="../asset/image/LOGO.svg" alt="logo">
             <nav>
                 <ul>
-                    <li><a href="index.html">Home</a></li>
-                    <li><a href="pages/about.html">About</a></li>
-                    <li><a href="pages/contact.html">Contact</a></li>
+                    <li><a href="../index.html">Home</a></li>
+                    <li><a href="../pages/about.html">About</a></li>
+                    <li><a href="../pages/Contact.html">Contact</a></li>
                     <li><a href="pages/blog.html">Blog</a></li>
+                    <li><a href="pages/Login.html">Login</a></li>
                     <li><a href="pages/sign.html">Sign up</a></li>
                 </ul>
             </nav>
-        </div>
-        <div class="body-container">
-            <div class="contact">
-                <h1>Contact me:</h1>
             </div>
-        </div>
\ No newline at end of file
+        <div class="body">
+         <div class="body-container">
+            <h1>Contact me:</h1>
+        </div>
+        <div class="form-container">
+        <form class="my form">
+            <label for="fname">First Name</label>
+            <input type="text" id="fname" name="firstname" placeholder="Your name..">
+        
+            <label for="lname">Second Name</label>
+            <input type="text" id="lname" name="lastname" placeholder="Your Second name..">
+
+            <label for="lname">Email</label>
+            <input type="text" id="lname" name="lastname" placeholder="Your Email..">
```

Comment:
You can not have the same ID (name) for more than 1 HTML element, IDs should be unique